src/deploy.ts

```diff
@@ -123,9 +124,9 @@ export class HypERC20Deployer extends GasRouterDeployer<
   ): Promise<HypERC20> {
     const router = await this.deployContractFromFactory(
       chain,
-      new HypERC20__factory(),
-      'HypERC20',
-      [config.decimals],
+      config.votable ? new HypERC20Votable__factory : new HypERC20__factory(),
+      config.votable ? 'HypERC20Votable' : "HypERC20",
+      config.votable ? [] : [config.decimals],
```

Comment:
If `HypERC20Votable` is just inherting from `HypERC20`, then they should have the same args?

Reply:
HypERC20Votable is inheriting the ERC20 contract which already has fixed decimals. So no need to initialize it as we do in normal HypERC20.